src/smefit/loader.py

```diff
@@ -602,18 +602,16 @@ def load_datasets(
     th_cov = []
 
     Loader.commondata_path = pathlib.Path(commondata_path)
-    if theory_path is not None:
-        Loader.theory_path = pathlib.Path(theory_path)
-    else:
-        Loader.theory_path = pathlib.Path(commondata_path)
+    Loader.theory_path = pathlib.Path(theory_path or commondata_path)
 
     _logger.info(f"Applying cutoff scale: {cutoff_scale} GeV.")
-    for sset in np.unique(datasets):
+    for sset in datasets:
```

Comment:
Here and in similar loops (the one in load_rge_mat for example) datasets are looped without being put in alphabetical order now? I am a bit worried about having mismatches, are we guaranteed that things are ordered correctly?

Reply:
I see your concern, but datasets is a list which has a well defined ordering, so there's no need to sort anything (as opposed to dictionaries). But it was a list also before, and yet we had it got sorted. Perhaps @giacomomagni can comment if/why this was necessary at the time?

Reply:
I guess the unique was there just to be sure a dataset is not loaded twice, but yes from the `DataTuple` you can always infer the order.